### ISSUE TITLE
[Bug]: Can't use System Wine / Vaniglia always broken

### ISSUE THREAD

**Issue by nPHYN1T3** [bottlesdevs/Bottles#1254]
### Describe the bug

I can no longer select system WINE and vaniglia never works. Normally because vaniglia (this is the only runner installed by default from the version in the repo's) has never worked I always switch to the system Wine however now while I can select it it never saves/updates and I get "ERROR Runner sys-wine-7.5 not found in $HOME/.local/share/bottles/runners/sys-wine-7.5" Why it's looking for the system wine there is beyond me.

If I check older bottles sys-wine says it's version 7.3 but under preferences for the bottle the listed runner is blank. The application will run if I launch it but no runner is "selected" in the drop down.

Trying to reselect and run anything with vaniglia just pops out
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -a -r" (2).
wine: could not load kernel32.dll, status c0000135

### To Reproduce

Have Bottles installed for a while, perhaps have had some updates? Try to use system wine...enjoy the nothing?

### Package

AUR ("bottles" package)

### Distribution

Garuda Arch

### Health Check

Display:
    X.org: true
    X.org (port): :0.0
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/share/vulkan/icd.d/nvidia_icd.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.1-arch1-1
Distro:
    Name: Garuda Linux
    Version: Unknown
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.35'
Bottles_envs: null

### Display Backend used by Bottles

X11

### Additional context

Bottles 2022.3.28-trento-1
Trying to change runners switching back to Vaniglia

**Comment by mirkobrombin** [comment]
Thanks for reporting. Our default runner is Caffe (currently 7.5). Btw I'm able to switch to the sys-wine runner and it works.

**Comment by nPHYN1T3** [comment]
Every time I've tried Bottles over the last few years Vaniglia is always set as default. I may nuke the whole install and see if a reinstall fixes things...or makes it worse.

**Comment by mirkobrombin** [comment]
Vaniglia was replaced by Caffe like 8 releases ago. The new one is based on TKG

**Comment by nPHYN1T3** [comment]
Lol well that begs the question then is the repo version old. I just renamed my configs, nuked the install and reinstalled to test and man this is broken. I have a welcome pop up that literally only renders part of the panel. WELCOME! What are runners...small dots that are clearly the next line of text...

**Comment by mirkobrombin** [comment]
![Screenshot from 2022-04-01 22-59-33](https://user-images.githubusercontent.com/9479903/161340671-07845846-96cf-4e5a-ab90-7d6c2843d2f4.png)
![Screenshot from 2022-04-01 22-59-35](https://user-images.githubusercontent.com/9479903/161340675-a0c16427-0ae8-43d4-83d0-b92790d5a55b.png)
![Screenshot from 2022-04-01 22-59-37](https://user-images.githubusercontent.com/9479903/161340679-bbf1bbd5-32be-4972-8b5a-bb9f84a97dfa.png)
![Screenshot from 2022-04-01 22-59-39](https://user-images.githubusercontent.com/9479903/161340680-ac6bde51-6dbe-40e3-a353-d9ac70accd36.png)
![Screenshot from 2022-04-01 23-00-11](https://user-images.githubusercontent.com/9479903/161340683-ca0c5ccc-6697-4a55-a095-4708a32d9c53.png)
![Screenshot from 2022-04-01 23-00-46](https://user-images.githubusercontent.com/9479903/161340684-3f6dc103-0760-44e2-8a7b-332a23139845.png)
![Screenshot from 2022-04-01 23-00-51](https://user-images.githubusercontent.com/9479903/161340685-cfbc530f-9687-4f37-b868-792dd4150b64.png)

**Comment by nPHYN1T3** [comment]
Yeah that's the welcome but it only renders up the the what are runners, then like I said I can see like 1px of the next line of text. I just click next until it does the sit back and relax bit...but then I try to make a bottle and it dies with the errors I posted above.

**Comment by mirkobrombin** [comment]
I was confused by:
> Version 2022.3.28-trento-1 --> still defaults to Vaniglia.

Btw I've made a new bottle and it also works as expected:

![Screenshot from 2022-04-01 23-08-44](https://user-images.githubusercontent.com/9479903/161341532-9d151c16-a6b4-42de-9f91-b3af69f92434.png)
![Screenshot from 2022-04-01 23-08-51](https://user-images.githubusercontent.com/9479903/161341534-99c5e131-156a-4ed6-8456-2ce719832dc6.png)

Idk, maybe I'm just not understanding :D

**Comment by nPHYN1T3** [comment]
Well "works for me" isn't really valid here. Not sure what you aren't understanding. The version in the repo's defaults to Vaniglia for a runner. I can't select the system wine, and on a clean install there are tons of problems (all noted above) leaving bottles completely broken. I tried the version in the chaotic aur, the welcome renders less of the panel and the git won't build
![0-April 01 2022 15:12:53](https://user-images.githubusercontent.com/38122105/161342011-839df2d3-d96e-4717-acfb-53aa291daa10.png)
.

**Comment by mirkobrombin** [comment]
I'm trying to understand and reproduce the problem in any of our official packages but I'm not able, really. I can't reproduce this and "works for me" is not my answer to your problem. 

I'm trying to help you but I can't reproduce the problem. Such DLL should be there and if not, mean that your bottle is just broken. I can't just push your bottle creation, so I'm raising an error. I've tested with Flatpak and AUR and it downloads Caffe and the bottle creation goes fine. Again, it's not "works for me" but "I can't reproduce".

**Comment by nPHYN1T3** [comment]
Yes well I can't reproduce either in that the initial issue was the system Wine was no longer selectable or working but upon trying a clean install things look far more broken.

If I was to spitball I'd wager this is probably an environment variable missing somewhere that bottles never complains about.

**Comment by mirkobrombin** [comment]
Can you try the flatpak? Just to exclude a package related problem. I’ll do another test tomorrow with chaotic in a vm

**Comment by nPHYN1T3** [comment]
Le FFS...I was going to say no on Flat or Snap so I went to look for an appImage...
"AppImage is broken, we are investigating, please use Flatpak in the meantime."

**Comment by mirkobrombin** [comment]
I remembered that Bottles checks if the glibc version is at least 2.32 because it is required by Caffe and if it does not find it folds on another runner. But I see you have 2.35 so I exclude this too

**Comment by nPHYN1T3** [comment]
I'm trying on a VM but it's being all cranky and needing updates before I can try to install. I may fire up an Ubuntu VM if the Arch one yields the same result and see if flat/snap on Ubuntu works. I still am kinda suspicious of some path or env issue. When I last installed bottles I was under a completely different environment. Now I'm in brutally stripped down "old school" environment that won't/doesn't have all the fancy crap.

**Comment by mirkobrombin** [comment]
Just one notice, Ubuntu has glibc < 2.32 so it will not use Caffe. Y default

**Comment by nPHYN1T3** [comment]
Heh I don't care which it defaults to so long as it works. This is kinda the crux of this since Bottles always defaulted to the V runner which never worked so I always just set it to use the system Wine version.

**Comment by mirkobrombin** [comment]
Flatpak came with glibc 2.33 which is fine enough to make it works (at least for testing purposes as you do not want to use flatpak)

**Comment by nPHYN1T3** [comment]
Running under Awsome and the UI rendered the welcome bit correctly...(update in a few)

**Comment by RushingAlien** [comment]
Just to confirm, did you also delete user data after reinstalling when testing on a fresh install?
for non-flatpak it would be in `~/.local/share/bottles/`

**Comment by nPHYN1T3** [comment]
Yeah I nuked the whole thing. Initially I just renamed it but eventually I just deleted the whole deal.

I tried to set the system Wine as the runner on the Garuda test VM and it also fails like on my workstation.

**Comment by RushingAlien** [comment]
Just tested the AUR version with sys-wine as runner.
![gambar](https://user-images.githubusercontent.com/80886145/161349988-2a9d9093-c785-4115-b24a-ba153fc0205f.png)
Seems to work fine, it would seem it's very difficult to reproduce you're issue.

**Comment by nPHYN1T3** [comment]
When the slow Azz Ubuntu install is done I may try with the deb version. I'm starting to wonder if this isn't something with Garuda explicitly since the Garuda VM I tested with also fails to set the system Wine.

**Comment by nPHYN1T3** [comment]
One last note. I don't know how I missed this but if you look on the one screenshot of the error in the list of "tasks completed" it says it's running as a flatpak despite flatpak not being on my system. Next is I again tested the chaotic AUR version against the normal AUR and both have the same issues. However I got curious about something.

$HOME/.local/share/bottles/dxvk/dxvk-1.10.1/x32/d3d9.dll -> $HOME/.local/share/bottles/bottles/test/drive_c/windows//syswow64/d3d9.dll
(20:17:56) ERROR Error while running async job: <bound method Manager.create_bottle of <bottles.backend.managers.manager.Manager object at 0x7f1c40fe2290>>
	Exception: [Errno 2] No such file or directory: '$HOME/.local/share/bottles/bottles/test/drive_c/windows//syswow64/d3d9.dll'

Outside the //syswow that made me think there was a syntax issue in there I tried to make a bottle and watch as it's being made.

So on a clean install I get the set up stuff.
========Downloading vaniglia-7.1-unstable-x86_64.tar.gz: 100 [==================================================>

(20:17:06) INFO Renaming [vaniglia-7.1-unstable-x86_64.tar.gz] to [vaniglia-7.1-unstable-x86_64.tar.gz]. 
(20:17:08) INFO Runners found:
	 - sys-wine-7.5
	 - vaniglia-7.1-unstable
(20:17:48) INFO Generating bottle configuration… 
(20:17:48) INFO Using WINE Runtime tool -- send_status(-i) 
/bin/sh: line 1: $HOME/.local/share/bottles/runners/vaniglia-7.1-unstable/bin/wine64: No such file or directory <-- uh did you just download this? (It never says it extracts it)

Inbetween registry entry actions it keeps spamming about the vaniglia not being there and eventually dies for good on the dxvk. This seems to be because as I watched things it makes the directory for the bottle but never actually puts anything in it. i.e. when it tries to copy over the dxvk dll's the target directory doesn't exist and that kills it. 

Checking the runners the vaniglia folder only has lib and lib64, no bin, include or share. When I check under preferences and runners it has the vaniglia-7.1-unstable listed as not installed but that is the file it said it was grabbing (and it did) and installing as the default. If I check the tgz in the temp folder it did downloading it but the zip is missing things. If I install the caffe then it actually makes the bottle.
 
So the clean install problems seems to be triggered by whatever makes bottles default to vaniglia and the fact the vaniglia tgz is missing things. As for the initial issue of not being able to set system Wine, that is "broken" on all my test machines as well. I quote broken because I don't know why the Flatpak version says it set to the system wine but it's not.

**Comment by mirkobrombin** [comment]
Unstable builds are installed only if the “unstable” filter is enabled. Btw I’ve just removed it from the repository

**Comment by nPHYN1T3** [comment]
When I reported this it was installing vaniglia "stable" (it didn't say otherwise). It wasn't until I nuked my settings and did ye olde pacman -R bottles ; pacman -S bottles that it started grabbing unstable. i.e. there were no settings to enable either as it was a clean install with no previous configs/settings to adhere to. I'm guessing the #1 issue is why it wants to go vaniglia when you said there IS a fall back but my libs shouldn't be triggering it. #2 would be that malformed tgz it grabs for vaniglia that then gibs everything after the fact. Then #3 why on my machine, VM of the same distro (different WM/DE), and the VM using flatpak they all fail to set system wine correctly.

**Comment by Lucki** [comment]
I also have this problem that the system wine can't be used:
~~~
ERROR Runner sys-wine-7.6 not found in /home/lucki/.local/share/bottles/runners/sys-wine-7.6
~~~
Installed is `wine-staging 7.6-1`.

This error isn't represented in the GUI. When creating the bottle or changing an existing bottle runner to `sys-wine-7.6` no indicator of an error is given anywhere. According to the GUI everything is fine.

I don't have that other problem with vanaglia being used instead of caffe.

<details>
<summary>Health Check</summary>

~~~
Display:
    X.org: true
    X.org (port): :1
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/share/vulkan/icd.d/nvidia_icd.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.17.3-arch1-1
Distro:
    Name: Arch Linux
    Version: Unknown
Tools:
    cabextract: true
    p7zip: true
    patool: true
    glibc_min: '2.35'
Bottles_envs: null
~~~

</details>

**Comment by mirkobrombin** [comment]
Fixed by 4005775797a6bd4f3cc0939ad99f9b1a3ab8d610
